coderd/notifications/testdata/rendered-templates/TemplateUserAccountDeleted-body.md.golden

```diff
@@ -1,3 +1,5 @@
 Hi Bobby,
 
-User account **bobby** has been deleted.
\ No newline at end of file
+User account **bobby** has been deleted.
+
+The deleted account belonged to **william tables** and was deleted by **rob**.
```

Comment:
Curious, if we can make notifications `TemplateUserAccountDeleted`, `TemplateUserAccountCreated`, `TemplateUserAccountActivated` and `TemplateUserAccountSuspended` sound more natural?

Any ideas, @dannykopping? Should we involve Christin?

Reply:
I'd be keen for a professional Wordsmith to have a look if we want.

Reply:
Agree, we should involve @EdwardAngert.

Reply:
I think we can do this in a follow-up. Let's get this merged.